### PR TITLE
Import lesson

### PR DESCRIPTION
**Pull request by ngodfraind**

### CODE REVIEW

**Comment by ngodfraind** [comment]
I don't know why travis went mad. Otherwise it looks good.

**Comment by stefk** [comment]
> I don't know why travis went mad.

PHPMD found an unused private method: https://travis-ci.org/claroline/Distribution/builds/127467561#L3124.

File: composer.json [hunk 1]
```diff
@@ -227,7 +228,8 @@
             "Claroline\\DevBundle\\ClarolineDevBundle",
             "Icap\\WebsiteBundle\\IcapWebsiteBundle",
             "Innova\\VideoRecorderBundle\\InnovaVideoRecorderBundle",
-            "Innova\\AudioRecorderBundle\\InnovaAudioRecorderBundle"
+            "Innova\\AudioRecorderBundle\\InnovaAudioRecorderBundle",
+            "Icap\\LessonBundle\\IcapLessonBundle"
         ]
```
Comment (stefk) [inline composer.json]: Should be added to the `replace` section as well.

File: plugin/lesson/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.idea
+VERSION.txt
```
Comment (stefk) [inline plugin/lesson/.gitignore]: This file can be removed.

**Comment by ngodfraind** [comment]
Can it display the unused methods ?

**Comment by ptsavdar** [comment]
LGTM

**Comment by stefk** [comment]
> Can it display the unused methods ?

No, a custom report renderer is required for that.

**Comment by stefk** [comment]
LGTM

**Pull request merged**